src/dvclive/live.py

```diff
@@ -607,14 +608,14 @@ def end(self):
         cleanup_dvclive_step_completed()
 
     def read_step(self):
-        if Path(self.metrics_file).exists():
-            latest = self.read_latest()
-            return latest.get("step", 0)
-        return 0
+        latest = self.read_latest()
+        return latest.get("step", 0)
 
     def read_latest(self):
-        with open(self.metrics_file, encoding="utf-8") as fobj:
-            return json.load(fobj)
+        if Path(self.metrics_file).exists():
+            with open(self.metrics_file, encoding="utf-8") as fobj:
+                return json.load(fobj)
+        return {}
```

Comment:
This was just refactoring so I could use `read_latest` elsewhere without having to check if the summary metrics file exists.